tools/shared.py

```diff
@@ -690,6 +690,7 @@ def get_llvm_target():
 LLVM_COMPILER = os.path.expanduser(build_llvm_tool_path(exe_suffix('llc')))
 LLVM_DWARFDUMP = os.path.expanduser(build_llvm_tool_path(exe_suffix('llvm-dwarfdump')))
 LLVM_OBJCOPY = os.path.expanduser(build_llvm_tool_path(exe_suffix('llvm-objcopy')))
+LLVM_OBJDUMP = os.path.expanduser(build_llvm_tool_path(exe_suffix('llvm-objdump')))
```

Comment:
If the tool is only needed during testing can you put it in tests/common.py (see WASM_DIS for example)

Reply:
Done.